include/MixerChannelView.h

```diff
@@ -38,6 +38,11 @@
 #include <QPixmap>
 #include <QWidget>
 
+namespace lmms
+{
+    class MixerChannel;
+}
```

Comment:
Could just write:
```suggestion
class lmms::MixerChannel;
```

Reply:
This did not work for me. With the changes the compiler expects a class definition.

Reply:
Hm, thought this would work. Maybe I'm thinking of a `using` statement or something else.